common/util.go

```diff
@@ -336,6 +341,10 @@ func IsServiceClientTransientError(err error) bool {
 }
 
 func IsServiceHandlerRetryableError(err error) bool {
+	if err.Error() == ErrNamespaceHandover.Error() {
+		return false
+	}
+
```

Comment:
Will this disable all retries, or some retries like between FE and History will still happen?

Reply:
We decided to remove all retries within the system for this error.

NOTE: frontend handler is also not retrying this error. <- behavior is not introduced by this pr, but a previous fix related to frontend interceptor (retry interceptor was moved after namespace state validator)

Reply:
Will SDK side retry on this error?

Reply:
Yeah I think so. SDK still sees Unavailable error.